src/LanguageServer.php

```diff
@@ -190,6 +191,7 @@ public function initialize(ClientCapabilities $capabilities, string $rootPath =
             $this->projectIndex = new ProjectIndex($sourceIndex, $dependenciesIndex, $this->composerJson);
             $stubsIndex = StubsIndex::read();
             $this->globalIndex = new GlobalIndex($stubsIndex, $this->projectIndex);
+            $initializationOptions = $initializationOptions ?? new Options;
```

Comment:
For `$initializationOptions` you have more freedom how and which options will be send.

The `$initializationOptions` parameter ist typed in `initialize()` and is only passed as is to `coroutine()`, so it should be enough only to check for `null`.